src/Generator/Passes/RenamePass.cs

```diff
@@ -422,7 +425,7 @@ public static string ConvertCaseString(Declaration decl, RenameCasePattern patte
                     break;
             }
 
-            return sb.ToString();
+            return CSharpSources.SafeIdentifier(sb.ToString());
```

Comment:
Unfortunately this code is C# specific, while this pass should be language-agnostic.

What ideally would happen is that we would have a safe identifier generic function to call here, but it seems like right now we only have some code to deal with this in `CheckKeywordNamesPass`: https://github.com/mono/CppSharp/blob/main/src/Generator/Passes/CheckKeywordNamesPass.cs#L28

Anyway, that pass runs unconditionally after user passes, as you can see here: https://github.com/mono/CppSharp/blob/main/src/Generator/Driver.cs#L298

So either we should rework the code and call a language-specific safe identifier method here or just remove this code and let the pass deal with it.